### PR TITLE
Update code to deal with v2 API for suppressions

### PR DESCRIPTION
**Pull request by marcosgm**

### CODE REVIEW

File: lw-suppressions-migration/migrate.py [hunk 1]
```diff
@@ -19,21 +24,34 @@
...
+class SuppressionsAPIOverride (SuppressionsAPI):
+    def __init__(self, session):
+        super().__init__(session)
+
+    def get(self,
+            type,
+            recommendation_id=None):
+        if recommendation_id:
+            logger.info(f"Getting {type} suppression {recommendation_id} from Lacework...")
+            api_uri = f"/api/v2/suppressions/{type}/allExceptions/{recommendation_id}"
```
Comment (rmoles) [inline lw-suppressions-migration/migrate.py:36]: FWIW I don't believe we can get an individual recommendations' suppressions. 
```
➜  go-sdk git:(main) ✗ lacework api get suppressions/aws/allExceptions/LW_AWS_RDS_1
Usage:
  lacework api <method> <path> [flags]

Flags:
  -d, --data string   data to send only for post and patch requests
  -h, --help          help for api

Global Flags:
  -a, --account string      account subdomain of URL (i.e. <ACCOUNT>.lacework.net)
  -k, --api_key string      access key id
  -s, --api_secret string   secret access key
      --api_token string    access token (replaces the use of api_key and api_secret)
      --debug               turn on debug logging
      --json                switch commands output from human-readable to json format
      --nocache             turn off caching
      --nocolor             turn off colors
      --noninteractive      turn off interactive mode (disable spinners, prompts, etc.)
      --organization        access organization level data sets (org admins only)
  -p, --profile string      switch between profiles configured at ~/.lacework.toml
      --subaccount string   sub-account name inside your organization (org admins only)

ERROR unable to send the request:
  [GET] https://uedemo.lacework.net/api/v2/suppressions/aws/allExceptions/LW_AWS_RDS_1
  [400] Unrecognized url: suppressions
```

Comment (rmoles) [reply lw-suppressions-migration/migrate.py:36]: Whereas `lacework api get suppressions/aws/allExceptions` works for getting all suppressions

**Review by marcosgm — COMMENTED** [review]
fixed

**Comment by marcosgm** [comment]
Merging to ensure script works with RBAC-enabled customers (v1 api is gone)

**Pull request merged**